content/changelog/2024/new-prompts-ui.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "New, sleeker prompts interface"
+authors:
+  - name: Alex Isken
+    url: https://x.com/AlexanderIsken
+    avatar: https://storage.googleapis.com/sourcegraph-assets/blog/alex_avatar.png
+publishDate: 2024-10-15T10:00-02:00
+description: "Cody has a new interface for quickly selecting prompts and commands from the sidebar."
+tags: [Cody, VS Code]
+version: [v1.38]
+versionIcon: /assets/changelog/vscode.svg
+slug: 'new-prompts-ui'
+published: true
```

Comment:
```suggestion
```
@iskyOS, can we please add the field for related topics? The field is empty on the preview.

![CleanShot 2024-10-16 at 10 20 12@2x](https://github.com/user-attachments/assets/37cd5bc8-c861-4dff-a0fd-28645c5ff024)

Reply:
thanks Maedah! Fixed